README.rst

```diff
@@ -166,35 +164,36 @@ And you can view all the atributes on the evaluation result
 Using with Jupyter Notebook (or without a django app)
 -----------------------------------------------------
 
-Django-Estimators can run as a standalone django app.In order to have access to the django db, you'll need to set up the environment variable to load up your django project.  In ipython, by default you can set the environment variable ``DJANGO_SETTINGS_MODULE`` to ``estimators.template_settings`` like so
+Django-Estimators can run as a standalone django app. In order to have access to the django db, set up the environment variable to load up your django project.  In ipython, set the environment variable ``DJANGO_SETTINGS_MODULE`` to ``estimators.template_settings``:
 ::
 
     import os
     import django
     os.environ['DJANGO_SETTINGS_MODULE'] = "estimators.template_settings"
     django.setup()
 
-If you're creating a new database (by default it's ``db.sqlite3``).  Therefore we need to run migrations, so in python
+When creating a new database (by default ``db.sqlite3``). Run this migration:
```

Comment:
Run migrations from the interpreter after creating a new database (by default ``db.sqlite3``).